app/models/metadata_presenter/previous_page.rb

```diff
@@ -0,0 +1,38 @@
+module MetadataPresenter
+  class PreviousPage
+    include ActiveModel::Model
+    attr_accessor :service, :user_data, :current_page, :referrer
+
+    def page
+      # what happens when a user enters in the middle of the flow
```

Comment:
can we add a test for this?

Reply:
Do you mean if the user jump pages? Then what should happen? Can you provide me with an example?

Reply:
user visits the start page, therefore there is a session. then they jump to the middle of a flow just because they know the URL for some reason.

should they go back to the start page as there are no answers? or should return to the previous page in the in the flow? Or just return to the referrer? am not sure what the correct behaviour should be

Reply:
Cool, that's a good question. I'll ask the team!

Reply:
Discussion on https://mojdt.slack.com/archives/CE78LKCU8/p1625144555035600
The conclusion is that we will create a specific card to handle the user jumping pages.